README.md

```diff
@@ -1,74 +1,177 @@
 # Tellor Data Specifications
 
-This repository contains data specifications for Query IDs. Query IDs are used as unique identifiers for data reported to and retrieved from the Tellor Oracle. In TellorX, `queryID` replaces the legacy `requestID` (tellor3 and earlier). Query ID is the common naming convention across this repository, the [oracle contracts](https://github.com/tellor-io/tellorX/blob/main/contracts/Oracle.sol) themselves, and the [Python telliot client](https://tellor-io.github.io/pytelliot/examples/legacy_query_example/).
+This repository contains the query specifications for interacting with the TellorX oracle.
 
-## Adding New ID's
+A Query specifies how to pose a question to the TellorX oracle, instructions for reporters on how to respond, including the format of the response, and any special dispute considerations.
 
-Make a PR in this repo to signal to reporters that the data is ready to reported and to show them how to assemble the corresponding queryID. 
+# Query Types
 
-### Fields:
+TellorX is designed to support arbitrary query types (`QueryType`).  A `QueryType` can have an arbitrary response type, specified by a structured
+ABI type string.  
 
-#### Question ID
+The following `QueryType`s are currently supported:
 
-    (name of file) can be guessed, but ultimately will be filled in by team.
+- LegacyRequest
```

Comment:
Why changed from `LegacyQuery`?

Reply:
I don't really care either way tbh, I can see the argument going both ways